integration_tests/features/Propagation.feature

```diff
@@ -50,3 +50,35 @@ Feature: Block Propagation
     Then I receive an error containing 'Orphan block'
     And all nodes are at height 1
 
+  Scenario: Nodes should never sync but keep insync
+    Given I have 2 seed nodes
+    And I have 4 base nodes connected to all seed nodes
+    And I have a base node MINER connected to all seed nodes
+    And I have a lagging delayed node LAG1 connected to node MINER with delay_lagging 10000
+    Given I have a lagging delayed node LAG2 connected to node MINER with delay_lagging 10000
+    When I wait 100 seconds
```

Comment:
Need a comment for the reason of this delay

Reply:
Left comment. If its not there the node misses all the messages